typescript/packages/reality-capture-analysis/src/integration-tests/RealityDataAnalysis.test.ts

```diff
@@ -9,12 +9,11 @@ import chaiAsPromised from "chai-as-promised";
 chai.use(chaiAsPromised);
 import path from "path";
 import * as dotenv from "dotenv";
-import { BentleyError } from "@itwin/core-bentley";
 import { RealityDataAnalysisService } from "../RealityDataAnalysisService";
-import { RealityDataTransferNode, ReferenceTableNode } from "@itwin/reality-data-transfer-node";
 import { ServiceAuthorizationClient } from "@itwin/service-authorization";
 import { JobState, RealityDataType } from "@itwin/reality-capture-common";
 import { O2DJobSettings } from "../Settings";
+import { RealityDataTransferNode, ReferenceTableNode } from "@itwin/reality-data-transfer";
```

Comment:
i was expecting this to be imported from a node subpath

ie

```
"@itwin/reality-data-transfer/node"
```

Reply:
Is it a problem? Node implements "browser", "node" and "default" condition branches. Those are not subpath, that's why we don't need "/node"

Reply:
so this works with node, but i dont believe this will work with webpack properly